### PR TITLE
CSP Header

### PR DESCRIPTION
**Pull request by SekharV**
1. Adding Content-Security-Policy Header
   for Github API call

### CODE REVIEW

File: HacktoberfestProject.Web/Services/TableService.cs [hunk 1]
```diff
@@ -52,18 +52,13 @@ public async Task<ServiceResponse<User>> GetUserAsync(string username)
...
-                    var temp = user.RepositoryPrAddedTo
+                    Repository repo = user.RepositoryPrAddedTo
                                    .FirstOrDefault(repo => repo.Owner.Equals(info.Owner, StringComparison.OrdinalIgnoreCase) &&
                                                            repo.Name.Equals(info.RepoName, StringComparison.OrdinalIgnoreCase));
-                    if (temp != null)
-                    {
-                        repo = temp;
-                    }
-                    else 
+                    if (repo == null)
```
Comment (rickvdbosch) [inline HacktoberfestProject.Web/Services/TableService.cs:61]: Cleaner. I like it.

File: HacktoberfestProject.Web/Startup.cs [hunk 2]
```diff
@@ -63,6 +63,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
             app.UseAuthorization();
 
+            app.Use(async (ctx, next) =>
+            {
+                ctx.Response.Headers.Add("Content-Security-Policy", "connect-src 'self' https://api.github.com;");
```
Comment (rickvdbosch) [inline HacktoberfestProject.Web/Startup.cs:68]: I think we should add a `default-src` as well:

>Your policy should include a `default-src` policy directive, which is a fallback for other resource types when they don't have policies of their own.

Source: [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)

**Review by rickvdbosch — CHANGES_REQUESTED** [review]

File: HacktoberfestProject.Web/Startup.cs (hunk 2 above)
Comment (Layla-P) [reply HacktoberfestProject.Web/Startup.cs:68]: sounds sensible. @SekharV are you okay to make the change?

Comment (SekharV) [reply HacktoberfestProject.Web/Startup.cs]: Done @Layla-P

**Review by rickvdbosch — APPROVED** [review]
looks like it's good to go!

**Pull request merged**